sherpa/astro/instrument.py

```diff
@@ -1279,7 +1279,7 @@ def create_arf(elo, ehi, specresp=None, exposure=None, ethresh=None, name='user-
 
 def create_delta_rmf(rmflo, rmfhi, offset=1,
                      e_min=None, e_max=None, ethresh=None,
-                     name='delta-rmf'):
+                     name='delta-rmf', header=None):
```

Comment:
If we do this then we should also do this for `create_non_delta_rmf` and `create_arf`.

Reply:
Am I right in thinking this comment can be closed with your new changes?

Reply:
yes, done.